src/pcre2_convert.c

```diff
@@ -188,7 +216,7 @@ while (plength > 0)
       switch (posix_state)
         {
         case POSIX_CLASS_STARTED:
-        if (c <= 127 && islower(c)) break;  /* Remain in started state */
+        if (ISLOWER(c)) break;  /* Remain in started state */
```

Comment:
Usually the comparison with 127 is present, because it is compatible with utf8. Changing it to 256 might have side effects.